pkg/ovsdb/ovsdb.go

```diff
@@ -43,7 +65,15 @@ type OvsBridgeDriver struct {
 func NewOvsDriver(ovsSocket string) (*OvsDriver, error) {
 	ovsDriver := new(OvsDriver)
 
-	ovsDB, err := libovsdb.ConnectWithUnixSocket(ovsSocket)
+	dbmodel, err := client.NewDBModel("Open_vSwitch",
+		map[string]client.Model{bridgeTable: &Bridge{}, ovsTable: &OpenvSwitch{}})
+	if err != nil {
+		return nil, fmt.Errorf("unable to create DB model error: %v", err)
+	}
+
+	_flag := flag.FlagSet{}
```

Comment:
A library function on this level should not interact with cli. If you need any extra parameters, we should read them in main.go and pass them down as function arguments

Reply:
Fixed to use ovsSocket argument directly.